src/main/scala/com/lightbend/conductr/sbt/ConductRPlugin.scala

```diff
@@ -45,26 +47,165 @@ object ConductrPlugin extends AutoPlugin {
+          case (Some(b), Some(bc)) =>
+            Def.taskDyn {
+              val bundleName = (normalizedName in b in projectRef).value
+              val bundleFile = (dist in b in projectRef).value.toPath
+              val bundleConfigContents = (BundleKeys.bundleConf in bc in projectRef).value
+              val srcDir = (sourceDirectory in bc in projectRef).value / (BundleKeys.configurationName in bc in projectRef).value
+              if (bundleConfigContents.nonEmpty || srcDir.exists()) {
+                Def.task {
+                  val bundleConfigFile = (dist in bc in projectRef).value.toPath
+                  Some(InstallationData(bundleName, Right(bundleFile), Some(bundleConfigFile))): Option[InstallationData]
+                }
+              } else {
+                Def.task[Option[InstallationData]] {
+                  Some(InstallationData(bundleName, Right(bundleFile), None))
+                }
+              }
+            }
+          case _ =>
+            Def.taskDyn(Def.task[Option[InstallationData]](None))
+        }
+      }
+
+    def fold(a: Seq[InstallationData])(tasks: Seq[Def.Initialize[Task[Option[InstallationData]]]]): Def.Initialize[Task[Seq[InstallationData]]] =
+      tasks match {
+        case Nil =>
+          Def.task(a)
+        case x :: xs =>
+          Def.taskDyn {
+            x.value match {
+              case Some(v) => fold(a :+ v)(xs)
+              case None    => fold(a)(xs)
+            }
+          }
+      }
+
+    fold(List.empty)(bundleInstallationScriptsTasks)
+  }
+
+  private def generateInstallationScriptTask(): Def.Initialize[Task[File]] = Def.task {
+    val installDir = (target in LocalRootProject).value
+    val installationScript = installDir / "install.sh"
+    val installPath = installDir.toPath
+    val installationScriptContents = installationData.value.map {
+      case InstallationData(bundleName, bundle, bundleConfigFile) =>
+        val bundleIdEnv = bundleName.replaceAll("\\W", "_").toUpperCase + "_BUNDLE_ID"
+        val bundleArg = InstallationData.nameOrPath(
+          bundle match {
+            case Left(b)  => Left(b)
+            case Right(b) => Right(installPath.relativize(b))
+          }
+        )
+        val bundleConfigArg = bundleConfigFile.fold("")(installPath.relativize(_) + " ")
+        s"""
+           |echo "Deploying $bundleName..."
+           |$bundleIdEnv=$$(conduct load $bundleArg $bundleConfigArg--long-ids -q)
+           |conduct run $${$bundleIdEnv} --no-wait -q
+                 """.stripMargin
+    }
+    IO.write(
+      installationScript,
+      installationScriptContents.mkString(
+        s"""#!/usr/bin/env bash
+            |cd "$$( dirname "$${BASH_SOURCE[0]}" )"
+            |""".stripMargin,
+        "",
+        """
+          |echo 'Your system is deployed. Running "conduct info" to observe the cluster.'
+          |echo 'Type just "conduct" for help.'
```

Comment:
Not sure if we need the line "echo 'Type just "conduct" for help.'". At that point in time the user should familiar with the conduct command. Also he might want to use `conduct logs`, depending on the situation.

If we stick with the line I would rename it to: `echo 'Type "conduct" for help.'`